spring-xd-starters/spring-xd-module-parent/publish-maven.gradle

```diff
@@ -162,7 +163,7 @@ def customizePom(pom, gradleProject) {
 				dependency {
 					groupId 'org.apache.hadoop'
 					artifactId 'hadoop-common'
-					version '2.3.0'
+					version '2.6.0'
```

Comment:
We probably could, but I would suggest creating a separate issue for it, as it's already stretching the scope of the original PR.